templates/drupal/1/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
+
+services:
+
+  drupal:
+    image: drupal:8.2-apache
+    labels:
+      io.rancher.sidekicks: postgres,drupal-datavolume
+      io.rancher.container.pull_image: always
+    ports:
+      - ${public_port}:80
+    links:
+      - postgres:postgres
+    volumes_from:
+      - drupal-datavolume
+    restart: always
+
+  postgres:
```

Comment:
Postgres shouldn't be a sidekick. Also, if you use "volumes_from", postgres would need another busybox  just for him.